coala_quickstart/generation/Utilities.py

```diff
@@ -0,0 +1,15 @@
+def comma_split(response):
+    """
+    Splits the comma-separated string into a list.
+
+    :param response:
+        Comma-separated string.
+    :return:
+        A list of comma-separated elements in the string.
+    """
+    result = []
+    for item in response.replace(" ", ",").split(","):
+        if len(item):
+            result.append(item)
+
+    return result
```

Comment:
you maybe want to use `StringConverter` in `coalib`, it also handles escaping and other stuff. As globs contain some special characters I'm not sure a plain comma (and especially space!) split work completely^^